nowcasting_api/tests/test_gsp.py

```diff
@@ -278,12 +278,154 @@ def test_read_truths_for_a_specific_gsp(db_session, api_client):
 
     app.dependency_overrides[get_session] = lambda: db_session
 
-    response = api_client.get("/v0/solar/GB/gsp/pvlive/122")
+    response = api_client.get("/v0/solar/GB/gsp/122/pvlive")
 
     assert response.status_code == 200
 
     r_json = response.json()
     assert len(r_json) == 3
+    # Also check the returned date format
+    assert r_json[0]["datetimeUtc"] == "2022-01-02T00:00:00Z"
+
+    _ = [GSPYield(**gsp_yield) for gsp_yield in r_json]
+
+
+@freeze_time("2022-01-02 12:10:00")
+def test_read_truths_for_specific_gsp_with_start_datetime(db_session, api_client):
+    """Check main solar/GB/gsp/{gsp_id}/pvlive route works with start_datetime_utc"""
```

Comment:
might be worth putting a link to [here](https://github.com/openclimatefix/nowcasting_datamodel/blob/main/tests/read/test_read_gsp.py) as this has a lot more tests in it